RUFAS/biophysical/animal/data_types/reproduction_inputs.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+
+from RUFAS.biophysical.animal.data_types.animal_enums import Breed
+from RUFAS.biophysical.animal.data_types.animal_events import AnimalEvents
+from RUFAS.biophysical.animal.data_types.animal_types import AnimalType
+
+
+@dataclass
+class ReproductionInputs:
+    animal_type: AnimalType
+    body_weight: float
+    breed: Breed
+    cull_reason: str
+    days_born: int
+    days_in_pregnancy: int
+    days_in_milking: int
```

Comment:
```suggestion
    days_in_milk: int
```
This keeps it consistent with the `GeneralProperties`.